File: src/main/java/hilfling/backend/hilfing/model/Gang.java
package hilfling.backend.hilfing.model;

import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;

@Data
@Entity
@Table(name = "gang")
public class Gang extends BaseEntity<Long>{
    public Gang() {
    }

    public Gang(String name) {
        this.name = name;
    }

    @Column(name = "name", nullable = false, unique = true)
    private String name;

}

File: src/main/java/hilfling/backend/hilfing/model/Gang.kt
package hilfling.backend.hilfing.model

import me.liuwj.ktorm.entity.Entity
import me.liuwj.ktorm.schema.Table
import me.liuwj.ktorm.schema.int


interface Gang : Entity<Gang> {
    companion object : Entity.Factory<Gang>()
    val id : Int
    val name: String
}

object Gans : Table<Gang>("gangs") {
    val id = int("id").primaryKey().bindTo {it.id}

}